js/arena.js

```diff
@@ -0,0 +1,58 @@
+class Arena {
```

Comment:
add `export default class Arena` and remove `export {Arena}`, it causes problems.